packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -8,25 +8,67 @@ import { View, Text, TouchableWithoutFeedback } from 'react-native';
  */
 import { __ } from '@wordpress/i18n';
 import { Dashicon } from '@wordpress/components';
+import { MediaUpload, MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO } from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
 import styles from './styles.scss';
 
 function MediaPlaceholder( props ) {
```

Comment:
We are making MediaPlaceholder usable by both image and video blocks by passing media type.